api/prerender.ts

```
@@ -0,0 +1,304 @@
+    .replaceAll('>', '&gt;')
+    .replaceAll('"', '&quot;')
+    .replaceAll("'", '&#39;')
+
+const stripMarkdown = (input: string) => {
+  let text = input || ''
+  text = text.replace(/```[\s\S]*?```/g, ' ')
+  text = text.replace(/`{1,3}[^`]*`{1,3}/g, ' ')
+  text = text.replace(/!\[[^\]]*]\([^)]+\)/g, ' ')
+  text = text.replace(/\[([^\]]+)]\([^)]+\)/g, '$1')
+  text = text.replace(/<[^>]+>/g, ' ')
+  text = text.replace(/[#>*_~\-]{1,}/g, ' ')
```

ESLint

[error] Unnecessary escape character: \-.

(no-useless-escape)